pythonAPIClient/client.py

```diff
@@ -3,25 +3,30 @@
 import requests
 
 class Client(object):
+
     """
     Represent a CAM2 client application.
     """
     # TODO: corresponding to the auth route
-    @staticmethod
-    def request_token(id, secret):
-        return 'dummy'
+    def request_token(self):
+        url = 'https://cam2-api.herokuapp.com/auth/?'
```

Comment:
Can we make the base url a class variable?  This is going to be used all over the place and could be changed in the future, so it'd be easiest to update it in one place than everywhere.

Something like: BASE_URL = 'https://cam2-api.herokuapp.com/'.

Reply:
done pls ignore